### ISSUE TITLE
Active classname for all visible elements

### ISSUE THREAD

**Issue by PiotrSzlagura** [brainhubeu/react-carousel#313]
**Is your feature request related to a problem? Please describe.**
I'm not 100% sure if it's a feature request, bug or is it already possible with current version.

**Describe the solution you'd like**
Right now if `slidesPerPage > 1`, only one item gets className `BrainhubCarouselItem--active`. Is there a way to add this className to every currently visible element? Or something else, like `BrainhubCarouselItem--visible` etc...

**Describe alternatives you've considered**
I tried working around it by altering classnames of children passed down to carousel and calculating (or assuming) when they are visible, but I think it's mixing responsibilities, so I dropped this idea.

**Additional context**

![Cały_ekran_10_03_2020__11_27](https://user-images.githubusercontent.com/15186961/76303640-57891300-62c2-11ea-8186-8a6a426a4c9b.png)

Thanks in advance for your help!

**Comment by piotr-s-brainhub** [comment]
@PiotrSzlagura 

It seems to be a bug.
Thanks for reporting.
Maybe we'll fix that but your PR will be more than welcome.

**Comment by PiotrSzlagura** [comment]
I'll try to approach this today :)

**Comment by piotr-s-brainhub** [comment]
@PiotrSzlagura 

We decided to improve your PR.

Or maybe you could do that but you need to consult that with @RobertHebel and @humbak .

**Comment by PiotrSzlagura** [comment]
I got a little bit lost with my assumptions and then forgot about this thread, sorry. Please tell me if my thinking is correct:

- If `slidesPerPage` is integer, then there is no partially visible items
- If `slidesPerPage` is float, then there are two partially visible items: one before first visible, and one after last visible item
- There can never be more than 2 partially visible items.

Am I right?

**Comment by piotr-s-brainhub** [comment]
@PiotrSzlagura 

IMO `slidesPerPage` should be always an integer.

**Comment by PiotrSzlagura** [comment]
@piotr-s-brainhub I don't agree. Having possibility to set non-integer slidesPerPage is common in sliders libs (intentional or not) and I personally am using it a lot, so I think it's good that it's there.

**Comment by piotr-s-brainhub** [comment]
@PiotrSzlagura 

So I'm OK with that, it can be a non-integer.
